JAICore/jaicore-ml/src/jaicore/ml/dyadranking/loss/DyadRankingLossFunction.java

```diff
@@ -0,0 +1,25 @@
+package jaicore.ml.dyadranking.loss;
+
+import jaicore.ml.dyadranking.dataset.IDyadRankingInstance;
+
+/**
+ * Loss function for evaluating dyad rankers.
+ * 
+ * @author Helena Graf
+ *
+ */
+public interface DyadRankingLossFunction {
```

Comment:
We might want to rename this, as our "losses" aren't technically losses but something like scores, i.e. higher is better.